src/App.vue

```diff
@@ -1,5 +1,9 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import HeaderComponent from './components/HeaderComponent.vue'
```

Comment:
Please add this code block in tsconfig.json in order to use import paths with '@/...'
"paths": { "@/*": [ "src/*" ] },